website/docs/community/bindings.md

```diff
@@ -32,6 +32,10 @@ As a C library, bindings can be made to call H3 functions from different program
 ## Clojure 
 
 - [Factual/geo](https://github.com/Factual/geo)
+- 
+## Dart 
```

Comment:
No, in current state the bindings can't be used outside of Flutter projects.
Although, it's not really difficult to make it work under Dart VM, need to reorganize structure a little, split the package on 2 different ones. 
So maybe it's temporary :)

I think 'Dart' title is more appropriate here than `Flutter`, because there's `JavaScript` section which contains 3 links - for pure JS, node JS and React Native. 
JS - RN, Dart - Flutter, similar.